### ISSUE TITLE
model consistency (missing field 'dilate')

### ISSUE THREAD

**Issue by marksunpeng** [vlfeat/matconvnet#755]
Hi guys,

I have some model(simplenn) trained with Version 20(or before). 
I try to use them with current version V23, and appears missing field 'dilate' in layers.

Do we have something simple like vl_simplenn_tidy to make all model consistent?

Thanks for your time, I'm currently working on it.

peng

**Comment by marksunpeng** [comment]
quick fix:

for i = 1:numel(net_old.layers)
if strcmp(net_old.layers{1,i}.type,'conv')
net_old.layers{1,i}.dilate = 1;
end
end

since 'dilate' is not an option for old version.

**Comment by omair-kg** [comment]
Haven't tried it but running the vl_simplenn_tidy function on the old net usually takes care of such backwards compatibility problems.

**Comment by marksunpeng** [comment]
@omair-kg 
yes, I agree. but for this issue. vl_simplenn_tidy is not helpful.

of course it is minor(fix easily), but sometimes annoying.

**Comment by lenck** [comment]
Hi, I don't understand, `vl_simplenn_tidy` seems to be taking care of the dilate field [link](https://github.com/vlfeat/matconvnet/blob/master/matlab/simplenn/vl_simplenn_tidy.m#L68)...

Or is there a bug in the `vl_simplenn_tidy` function?

**Comment by crisb-DUT** [comment]
Hi, I have met the same problem, 'missing field 'dilate''. I couldn't understand your quick fix, could you please describe it more  detailed? Thank you! @marksunpeng

**Comment by Sheofeen** [comment]
version older than beta23 can modify the vl_simplenn_tidy referring that one of beta23, filling missing data
